knetstat.c

```diff
@@ -31,6 +31,12 @@
 
 #include <net/net_namespace.h>
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(3,10,0)
```

Comment:
I think the code doesn't build on Linux versions before 3.10.0 anyway, so this part actually needed.